src/interpolations.jl

```diff
@@ -1,47 +1,73 @@
+        return new{shape, IP}(base)
     end
 end
 
-function InterfaceCellInterpolation(ip::ScalarInterpolation)
-    return InterfaceCellInterpolation(ip, ip)
+Ferrite.getnbasefunctions(ip::InterfaceCellInterpolation) = 2*getnbasefunctions(ip.base)
+
+Ferrite.adjust_dofs_during_distribution(ip::InterfaceCellInterpolation) = Ferrite.adjust_dofs_during_distribution(ip.base)
+
+Ferrite.n_components(ip::InterfaceCellInterpolation) = n_components(ip.base)
+
+Ferrite.getorder(ip::InterfaceCellInterpolation) = getorder(ip.base)
+Ferrite.getorder(ip::VectorizedInterpolation{<:Any,<:Any,<:Any,<:InterfaceCellInterpolation}) = getorder(ip.ip)
+
+function Ferrite.vertexdof_indices(ip::InterfaceCellInterpolation)
+    here  = Ferrite.vertexdof_indices(ip.base)
+    there = map(v -> map(d -> d + nvertexdofs(ip.base), v), here)
+    return (here..., there...)
 end
 
-Ferrite.getnbasefunctions(ip::InterfaceCellInterpolation) = getnbasefunctions(ip.here) + getnbasefunctions(ip.there)
+function Ferrite.edgedof_interior_indices(ip::InterfaceCellInterpolation{<:AbstractRefShape{3}})
+    basedofs = Ferrite.facedof_interior_indices(ip.base)
+    offset = _nvertexdofs(ip.base)
+    here  = map(v -> map(d -> d + offset, v), basedofs)
+    offset += _nfacedofs(ip.base)
+    there = map(v -> map(d -> d + offset, v), basedofs)
+    return (here..., there...)
+end
 
-"""
-    get_n_dofs_on_side(get_dofs::Function, ip::InterfaceCellInterpolation, side::Symbol)
+function Ferrite.facedof_interior_indices(ip::InterfaceCellInterpolation{<:AbstractRefShape{dim}}) where {dim}
+    basedofs = Ferrite.celldof_interior_indices(ip.base)
+    offset = _nvertexdofs(ip.base) + (dim == 3 ? _nedgedofs(ip.base) : 0)
+    here  = map(v -> map(d -> d + offset, v), basedofs)
+    offset += ncelldofs(ip.base)
+    there = map(v -> map(d -> d + offset, v), basedofs)
+    return (here, there)
+end
 
-Return the number of DOFs on a `side` (`:here` or `:there`) of an `InterfaceCellInterpolation`.
-The function `get_dofs` specifies which DOFs are considered, e.g. by passing `vertexdof_indices`.
-"""
-function get_n_dofs_on_side(get_dofs::Function, ip::InterfaceCellInterpolation, side::Symbol)
-    baseip = getproperty(ip, side)
-    return length(get_dofs(baseip)) > 0 ? sum(length.(get_dofs(baseip))) : 0
+function Ferrite.default_interpolation(::Type{InterfaceCell{shape, C}}) where {shape<:AbstractRefShape, C} 
+    return InterfaceCellInterpolation(default_interpolation(C))
 end
-function get_n_dofs_on_side(get_dofs::Function, ip::VectorizedInterpolation{<:Any,<:Any,<:Any,<:InterfaceCellInterpolation}, side::Symbol)
-    baseip = getproperty(ip.ip, side)
-    return length(get_dofs(baseip)) > 0 ? sum(length.(get_dofs(baseip)))*n_components(ip) : 0
+
+function Ferrite.default_geometric_interpolation(ip::InterfaceCellInterpolation{<:AbstractRefShape{sdim}, IP}) where {sdim, IP<:ScalarInterpolation}
+    return InterfaceCellInterpolation(ip.base)^sdim
+end
```

Comment:
```suggestion
function Ferrite.default_geometric_interpolation(ip::InterfaceCellInterpolation{<:AbstractRefShape{sdim}, IP}) where {sdim, IP<:ScalarInterpolation}
    return VectorizedInterpolation{sdim}(InterfaceCellInterpolation(ip.base))
end
```
Should analogously to [the Ferrite definition](https://github.com/Ferrite-FEM/Ferrite.jl/blob/a471823c2afcb7c8a354c135bce581d1ff303fa5/src/FEValues/CellValues.jl#L34-L36) remove the need for the extra definition below. (Since `ip.base` should always be a `ScalarInterpolation`.)